### ISSUE TITLE
[TTPReq] Education: How to accept Apple Pay and other digital wallets with Tap to Pay on iPhone

### ISSUE THREAD

**Issue by staskus** [woocommerce/woocommerce-ios#14322]
Designs & Copy: pdfdoF-5KF-p2
Flows: pdfdoF-5GF-p2
Depends on: https://github.com/woocommerce/woocommerce-ios/issues/14159

## Education Step Screen Functionality

If it's not already done, I suggest creating a reusable full-screen scroll view to fit as much educational content as needed. Support text blocks, or numbered text blocks as we have in 'Set up Tap to Pay'.

## How to accept Apple Pay and other digital wallets with Tap to Pay on iPhone

- Use the visual assets from [Marketing Guidelines](https://developer.apple.com/tap-to-pay/marketing-guidelines/)
- Use the latest bezels
- Use localized images for all the supported TTP markets

## Design

![Image](https://github.com/user-attachments/assets/6eeb6f29-d33b-42c1-a30d-28dc0c36d368)

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
